training/model_management/environments/model-management/context/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{latest-image-tag}}
```

Comment:
Is there a simpler base image you could use? This one will be pretty large.

Reply:
This is currently just used by download component and does not require pytorch as such. But if we are to add mlflow convertor component, this env will be needed. Can you share if we should create separate env for mlflow convertor component and keep it simple for downloader?

Reply:
It's up to you. It sounds like the lowest common denominator is the base image you're using now so you can stick with that.